test_requirements/django_16.txt

```diff
@@ -0,0 +1,5 @@
+-r base.txt
+Django>=1.6.9,<1.7
+django-discover-runner
```

Comment:
don't think that you need discover runner,  should be already a part of Django itself.